simapp/app.yaml

```diff
@@ -22,6 +22,8 @@ modules:
         - account: attribute
         - account: marker
           permissions: [ burner, minter ]
+        - account: mint
+          permissions: [ minter ]
```

Comment:
Is this directly needed or is it ok to transitively use via the marker module?

Reply:
I needed to add this to get the simulation tests working.  I can double check why and if there is some work around.

Reply:
That’s interesting. Perhaps the check requires the parent module to have all the permissions as well as the underlying one? If that’s the case you would need burn as well